# <center>Day20<center>

In [15]:
#Read this url and find the 10 most frequent words. romeo_and_juliet = 'http://www.gutenberg.org/files/1112/1112.txt'
romeo_and_juliet = 'http://www.gutenberg.org/files/1112/1112.txt'
import requests
response = requests.get(romeo_and_juliet)
http_code=response.status_code
if http_code >100 and http_code<199:
    print('Error:Informational')
elif http_code >200 and http_code<299:
    print('Success')
elif http_code >300 and http_code<399:
    print('Error:Redirection')
elif http_code >400 and http_code<499:
    print('Error:Client error')
elif http_code >400 and http_code<499:
    print('Error:Server error')



Error:Client error


In [14]:
url = 'https://api.thecatapi.com/v1/breeds'
reponse = requests.get(url)
response.status_code
########################################
url = 'https://archive.ics.uci.edu/ml/datasets.php'
reponse = requests.get(url)
response.status_code

404

Most of the url are not working response status is giving 404 but for the, such practice have been done in day 19 by geting the data using request get

# <center> Day20 Completed!!!</center>